### DISCUSSION TITLE
What conditions do I need to be an Github campus expert?

### DISCUSSION THREAD

**Discussion by WongiPark0628** [community/community#45348] [GitHub Education] [answered]
I would like to apply for github campus expert 2023 years, so I would appreciate it if you could let me know in advance.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @kalelpark 

The conditions are as shown below. You can learn more at this link: https://education.github.com/experts

![image](https://user-images.githubusercontent.com/47532703/214603738-8ba9c84f-afab-46f0-8e1d-6cc251036367.png)

**Reply by WongiPark0628** [reply]
Thank you for letting me know when I couldn't find the content. Thanks!!
